### DISCUSSION TITLE
Aggiornare lo stato di un sensore

### DISCUSSION THREAD

**Discussion by alessiomanzi** [indomus/forum#1968] [GENERALE - Discussioni generiche]
Buongiorno a tutti , scusate ma sono alle prime armi , ho inserito nella dashboard la scheda "Batteria telefono" non che mi serva , giusto per cominciare a fare un pò di pratica. Noto però che i valori si aggiornano molto lentamente. Come fare per aggiornare più frequentemente lo stato di questo sensore?
Grazie a tutti anticipatamente
Alessio

**Comment by snakuzzo** [comment]
Leggi qui ... Ultimo paragrafo

https://indomus.it/formazione/shelly-mqtt-e-http-comandi-utili/
